<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSettingsMineSweeper</class>
 <widget class="QWidget" name="FormSettingsMineSweeper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1" colspan="2">
      <widget class="QSlider" name="sliderPieceSize">
       <property name="minimum">
        <number>16</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
       <property name="pageStep">
        <number>8</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelPixels">
       <property name="text">
        <string>pixels</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QFrame" name="frameCellSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>69</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>69</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelCells">
       <property name="text">
        <string>cells</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cell size    </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Mines</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMineCount">
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEasy">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Easy</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupMineRatio</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNormal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Normal</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupMineRatio</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHard">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Hard</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupMineRatio</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMax">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Max</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupMineRatio</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAutoLock">
     <property name="text">
      <string>Autolock explicit mines.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonGroupMineRatio</sender>
   <signal>buttonClicked(QAbstractButton*)</signal>
   <receiver>FormSettingsMineSweeper</receiver>
   <slot>onRatioButtonClicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderPieceSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormSettingsMineSweeper</receiver>
   <slot>updateLabels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderPieceSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormSettingsMineSweeper</receiver>
   <slot>updateMineMax()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderPieceSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormSettingsMineSweeper</receiver>
   <slot>updateSubFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRatioButtonClicked(QAbstractButton*)</slot>
  <slot>updateBlockPiece(int)</slot>
  <slot>updateLabels()</slot>
  <slot>updateMineMax()</slot>
  <slot>updateSubFrame()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroupMineRatio"/>
 </buttongroups>
</ui>
